aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -162,17 +194,26 @@ def validate_inputs(self):
             self.report('{}'.format(exception))
             return self.exit_codes.ERROR_INVALID_INPUT_PSEUDO_POTENTIALS
 
-    def should_run_init(self):
+    def set_max_seconds(self, max_wallclock_seconds):
```

Comment:
This method is almost identical to method `_prepare_process_inputs`: can we merge them?

Reply:
I just forgot to remove `_prepare_process_inputs` as it is no longer used